api/v1alpha1/backend_types.go

```diff
@@ -187,8 +187,18 @@ type BackendTLSSettings struct {
 	// must be specified with at least one entry for a valid configuration. Only one of
 	// CACertificateRefs or WellKnownCACertificates may be specified, not both.
 	//
+	// Only used for DynamicResolver backends.
```

Comment:
I dont think will work, we'll need to enable this TLS section for all backend, lets raise a GH issue to track it